saleor/checkout/utils.py

```diff
@@ -16,6 +16,12 @@
 from ..account.forms import get_address_form
 from ..account.models import Address, User
 from ..account.utils import store_user_address
+from ..checkout.calculations import (
+    get_checkout_line_total,
```

Comment:
Mr. Wolski(@krzysztofwolski) - What do you think about removing `get_`?

Reply:
I kind of like the idea, lets roll with it

Reply:
@koradon let me know what do you think about the change

Reply:
A little bit late but I agree with @korycins